### PR TITLE
Affix sidebar closes #37

### LINKED ISSUE #37: Float menu sidebar on page when scrolling down

### PR DESCRIPTION
**Pull request by mikedao**
I had to do some black magic to get this to work. CSS is very much not my bag. 

Got here with the help of Corey and Bob Gu.

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -12,3 +12,11 @@ a.anchor {
...
   visibility: hidden;
 }
+
+div#floater {
+  float: left;
+  position: fixed;
+  max-width: 250px;
+}
+
+
```
Comment (EmilyMB) [inline app/assets/stylesheets/application.css]: Extra line?

File: db/seeds.rb [hunk 2]
```diff
@@ -14,7 +14,7 @@ def initialize
   def generate_categories
     @categories = Category.create([
       { name: "Specialty Pizzas" },
-      { name: "Submarine Sandwiches" },
+      { name: "Sub Sandwiches" },
```
Comment (EmilyMB) [inline db/seeds.rb:17]: :+1:

**Comment by EmilyMB** [comment]
Like it! Could just explain a little more about what you were doing besides "black magic"? (i.e. a few more words besides "affix sidebar").

**Comment by mikedao** [comment]
When I say black magic, i got the thing to hold still, but it looked all weird because i couldnt figure out sizing. Our menu was just too wide. So I changed submarine to sub, and it looks awesome now. Black Magic.

**Pull request merged**